build.gradle

```diff
@@ -48,3 +53,43 @@ artifacts {
     archives sourcesJar
     archives javadocJar
 }
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            groupId = project.group
+            artifactId = rootProject.name
+            version = project.version
+            pom {
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                    license {
+                        name = 'MIT License'
+                        url = 'http://www.opensource.org/licenses/mit-license.php'
+                    }
+                }
+            }
+            from components.java
+            artifact sourcesJar
+            artifact javadocJar
+        }
+    }
+}
+
+bintray {
+    user = findProperty("bintrayUser")
+    key = findProperty("bintrayApiKey")
+    publish = true
+    setPublications("mavenJava")
+    pkg {
+        userOrg = 'consensys'
+        repo = 'pegasys-repo'
+        name = 'noise-java'
+        licenses = ['Apache-2.0']
+        vcsUrl = 'https://github.com/ConsenSys/noise-java'
```

Comment:
In teku, we have some extra options here (version, description, website, issue tracker), should we add any of these?:  https://github.com/ConsenSys/teku/blob/e662f14682932ea0f5c51d15b3904eefd3488e92/build.gradle#L54-L68

Reply:
`version` is included to the publication automatically
`description, website, issue tracker` - I added them manually to the Bintray packet, but may be it makes sense to add them to Maven descriptor as well